docs/signals/configuration/views-services/index.md

```diff
@@ -1,21 +1,65 @@
 ---
-title: "Views and Services"
+title: "Attribute groups: views and services"
 sidebar_position: 30
-sidebar_label: "Views and Services"
+sidebar_label: "Attribute groups"
 ---
 
-To access attributes, you need to define a `View` and optionally group multiple views into a `Service`. This guide explains how to define views and services, apply them to your deployment, and retrieve calculated values.
+Signals has two attribute groupings:
+* Views, for defining attributes
+* Services, for consuming attributes
 
+Each view is a versioned collection of attributes, and specific to one entity and one data source (stream or batch).
 
-### What is a view?
-A `View` is a versioned collection of attributes grouped by a common `Entity` (e.g., `session_id` or `user_id`). Once defined, a `View` allows you to retrieve the calculated values of the attributes it contains.
+Services are groups of views. They provide a stable interface to use in your applications: by pinning specific view versions, they provide a consistent set of consumable attributes. Services can contain views from different entities, and different sources.
 
-### What is a service?
-A `Service` is a collection of views that streamlines the retrieval of multiple views. By grouping related views into a `Service`, you can efficiently manage and access user insights, making it easier to personalize applications and analyze behavior.
+By using services you can:
+* Iterate on attribute definitions without worrying about breaking downstream processes
+* Migrate to new view versions by updating the service definition, without having to update the application code
 
+## Attribute management
 
-### Defining a view
-You can define a `View` by passing in a list of previously defined attributes. Here's an example:
+TODO this is very confusing
+
+Every view has a version number. If you change the attributes of a view, increase the version number.
```

Comment:
Just noting that we need to expand on this – you can also change attributes of a view without changing the version number, the question is when to change the version number (what is a breaking change). We have a task in the backlog on this.